package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "wp-amp-theme",
+  "version": "0.1.0",
+  "description": "WordPress AMP Theme",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
@ThierryA This could have `DIFF_BASE=master ./node_modules/wp-dev-lib/pre-commit`.

Secondly, we could also use [husky](https://github.com/typicode/husky) to define and run the `pre-commit` hook as an npm script which is configured during `npm install`.

Reply:
Added `DIFF_BASE=develop ./node_modules/wp-dev-lib/pre-commit`, everything should be checked against the `develop` branch.

Regarding husky, I really like the idea for internal project but I am not in favor of enforcing pre-commit hook (specifically the dev-lib hook) for open source project because we cannot predict all user local environments, but we can predict Travis environment. Should the pre-commit hook fail to run on user local environment, we could loose valuable contribution and grow frustration even though the code they want to push would pass the build.